flink-streaming-java/src/main/java/org/apache/flink/streaming/api/environment/StreamExecutionEnvironment.java

```diff
@@ -1498,10 +1499,12 @@ public JobExecutionResult execute() throws Exception {
 	 *
 	 * @param jobName
 	 * 		Desired name of the job
+	 * 	@param detached
+	 * 		Whether to run the job in detached mode
 	 * @return The result of the job execution, containing elapsed time and accumulators.
 	 * @throws Exception which occurs during job execution.
 	 */
-	public abstract JobExecutionResult execute(String jobName) throws Exception;
+	public abstract JobExecutionResult execute(String jobName, boolean detached) throws Exception;
```

Comment:
this is a public method that you mustn't change; you have to add a completely new method.